Source/bmalloc/libpas/src/libpas/pas_commit_mode.h

```diff
@@ -32,7 +32,7 @@ PAS_BEGIN_EXTERN_C;
 
 /* pas_commit_mode is a named "is committed" boolean. */
 
-enum pas_commit_mode {
+enum pas_commit_mode : unsigned int {
```

Comment:
This is C++ one, and libpas is C.

Reply:
Removed and fixed this problem another way - though this is valid in C23 whenever WebKit upgrades to that.